pkg/stacks/templates/engines/helm2.go

```diff
@@ -0,0 +1,268 @@
+/*
+Copyright 2019 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package engines
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+
+	"github.com/ghodss/yaml"
+	"github.com/go-logr/logr"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/crossplaneio/crossplane-runtime/pkg/logging"
+	"github.com/crossplaneio/crossplane-runtime/pkg/meta"
+
+	"github.com/crossplaneio/crossplane/apis/stacks/v1alpha1"
+)
+
+// Helm2EngineRunner creates resources from files which are specified in the helm2 format
+type Helm2EngineRunner struct {
+	Log logr.Logger
+}
+
+const (
+	spec             = "spec"
+	valuesYaml       = "values.yaml"
+	engineImageName  = "crossplane/resource-engine-helm2:latest"
```

Comment:
i don't think this is going to work with an official release, because there is no `latest` tag that gets published to docker hub.  Did you think through how we may be able to at runtime just reuse the same version/tag that the main crossplane image is using?

Reply:
Dang. I was originally planning on addressing this in a future change, but if we don't publish the `latest` tag, that probably won't work very well : ). Do you know whether we always publish `master`? That seems like it'd be the simplest workaround for now

Reply:
yes, we do always publish a `master` tag, so that could be a short-term workaround.

Reply:
kk - awesome! I'm going to go with that for now then, but I've got a higher-quality fix on the list of hardening improvements to do.

Reply:
This should be done now